pkgs/development/tools/language-servers/coq-lsp/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchurl, fetchFromGitHub, makeBinaryWrapper, ocamlPackages, coqPackages }:
+
+ocamlPackages.buildDunePackage rec {
+  pname = "coq-lsp";
+  version = "0.1.3+v8.16";
```

Comment:
`+v8.16` refers to the Coq version right? Should this package inherit Coq version from attributes like `coqPackages_8_16` instead?